backend/process/src/http/routes.js

```diff
@@ -29,5 +29,21 @@ router.get("/health", (_, res) => {
     res.status(200).json({ games: 0 });
   }
 });
+  
 
+router.get("/restart", async (req,res) => {
+  const roomID = req.query.roomID
+  let game = Object.values(games).find(game => game.roomID === roomID);
+  if (game) {
```

Comment:
Games are keyed by their roomID so you can just do games[roomID]

Reply:
good catch, will change it